core/store/tx_manager_test.go

```diff
@@ -22,30 +24,35 @@ import (
 
 func TestTxManager_CreateTx_Success(t *testing.T) {
 	t.Parallel()
-	app, cleanup := cltest.NewApplicationWithKey(t)
+
+	store, cleanup := cltest.NewStore(t)
 	defer cleanup()
-	store := app.Store
-	manager := store.TxManager
 
-	from := cltest.GetAccountAddress(t, app.GetStore())
+	ctrl := gomock.NewController(t)
```

Comment:
Do we ever need `ctrl` outside of NewMockEthClient? Wondering if it'd be worth saving a line of boilerplate with a test helper.

Reply:
You need to call `ctrl.Finish` wherever you use this and it's now used in a bunch of tests other than the MockEthCallerSubscriber so it wouldn't make much of a difference.